app/controllers/admin/users_controller.rb

```diff
@@ -471,7 +471,6 @@ def invite_admin
     end
 
     user.active = true
-    user.approved = true
```

Comment:
do we need an extra test somewhere to confirm that if there is no reviewable the user still gets approved?

Reply:
This is already covered by the `users_controller_spec`. We could also add an additional test for the user class.